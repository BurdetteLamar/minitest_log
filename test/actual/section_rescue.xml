<log>
  <summary verdicts='1' failures='1' errors='1'/>
  <section name='rescue'>
    <rescued_exception>
      <class>
        RuntimeError
      </class>
      <message>
        Rescued!
      </message>
      <backtrace/>
    </rescued_exception>
  </section>
  <section name='no_rescue'>
    <uncaught_exception timestamp='0000-00-00-xxx-00.00.00.000' class='RuntimeError'>
      <message>
        Not rescued!
      </message>
      <backtrace>
      </backtrace>
    </uncaught_exception>
    <verdict method='verdict_assert_equal?' outcome='failed' id='__error_count__'>
      <exp_value>
        0
      </exp_value>
      <act_value>
        1
      </act_value>
      <exception class='Minitest::Assertion'>
        <message>
          Expected: 0 Actual: 1
        </message>
        <backtrace/>
      </exception>
    </verdict>
  </section>
</log>
