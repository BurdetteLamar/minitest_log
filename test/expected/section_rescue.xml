<log>
  <summary_ verdicts='0' failures='0' errors='1'/>
  <section_ name='rescue'>
    <rescued_exception_ class='RuntimeError' message='Rescued!'>
      <backtrace_/>
    </rescued_exception_>
  </section_>
  <section_ name='no_rescue'>
    <uncaught_exception_ timestamp='0000-00-00-xxx-00.00.00.000' class='RuntimeError'>
      <message_>
        Not rescued!
      </message_>
      <backtrace_>
      </backtrace_>
    </uncaught_exception_>
  </section_>
</log>
